File: src/sdk-reference/java/1/server/get-stats/snippets/get-stats.java
try {
    String stats = new String(kuzzle.getServer().getStats(
      1234567890000L,
      1541426610000L
    ));
    System.out.println("Kuzzle Stats as JSON string: " + stats);
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}

File: src/sdk-reference/go/1/server/get-stats/snippets/get-stats.go
start := time.Date(2001, time.September, 9, 1, 46, 40, 0, time.UTC)
stop := time.Now()

_, err := kuzzle.Server.GetStats(&start, &stop, nil)

if err != nil {
  log.Fatal(err)
} else {
  fmt.Println("Success")
}
